pyproject.toml

```diff
@@ -7,7 +7,7 @@ authors = ["Matthew Jones <matburt@redhat.com>",
 
 [tool.poetry.dependencies]
 python = "^3.6"
-pyyaml = "^3.12"     # Do not bump this version, its the minimum for el7 and el8 packaging
```

Comment:
@Spredzy should this say "maximum for EL7 and EL8 packaging"?

seems pretty clear that this PR should be closed.

Reply:
I will change the wording and yes, this one can be closed.